File: app/src/main/java/com/example/tutorial/activities/CircleActivity.java
package com.example.tutorial.activities;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.example.tutorial.ui.MovementView;

public class CircleActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(new MovementView(this));
    }
}

File: app/src/main/java/com/example/tutorial/activities/CircleActivity.kt
package com.example.tutorial.activities

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.tutorial.ui.MovementView

class CircleActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(MovementView(this))
    }
}